.pre-commit-config.yaml

```diff
@@ -99,8 +99,6 @@ repos:
       - id: markdownlint
         args: ["--fix"]
 
-  # Run tools on Jupyter notebooks
```

Comment:
This is a bit confusing, because there are actually no sections in this file, so better to remove this.